_source/_posts/2023-07-31-workflows-workshop.md

```diff
@@ -0,0 +1,939 @@
+
+The email report looks like this: 
+
+{% img blog/workflows-workshop/Workflows_email_report.jpg alt:"Todo email report showing each todo item from the Todo application with a boolean indicating completed state" width:"800" %}{: .center-image }
+
+|Posts in the enterprise-ready workshop series|
+| --- |
+| 1. [How to get Going with the Enterprise-Ready Identity for SaaS Apps Workshops](/blog/2023/07/27/enterprise-ready-getting-started) |
+| 2. [Enterprise-Ready Workshop: Authenticate with OpenID Connect](/blog/2023/07/28/oidc_workshop) |
+| 3. [Enterprise-Ready Workshop: Manage Users with SCIM](/blog/2023/07/28/scim-workshop) |
+| 4. [Enterprise-Ready Workshop: Terraform](/blog/2023/07/028/terraform-workshop) |
+| 5. **Enterprise-Ready Workshop: Automate with no-code Okta Workflows** |
+
+In a later section, you will learn how to get the Todo application. For now, you will learn about Okta Workflows. 
+
+
+## What is Okta Workflows?
+[Okta Workflows](https://www.okta.com/platform/workflows/) is a no-code platform that allows building automations to help with identity processes. 
+
+The following are use cases where you can use Workflows. 
+
+### Provision and deprovision app accounts
+When an employee joins your company, Workflows simplifies the task of provisioning their account.
+
+* Automatically create their identity in your apps
+* Set user and group entitlements
+* Assign shared folder
+* Send a message to their manager or a welcome message to the team Slack channel
+
+Similarly, when an employee leaves the company, Workflows can deactivate the user account, transfer their digital assets to a manager, and then deactivate the user account three days later.
+
+### Sequence actions with logic and timing
+Workflows can create non-activated accounts in all apps one week before a new employee's start date and then activate them on their first day.
+
+If an employee leaves your company, Workflows can deactivate the user account, remove their access to all apps except payroll, and then delete the account after a year.
+
+### Send notifications for lifecycle events
+For a lifecycle event such as an app assignment or user suspension, Workflows can notify your IT team through email or Slack.
+
+### Log and share lifecycle events
+Workflows can query Okta APIs and System Log events, run logic, and even compile data into a CSV file. Then, Workflows can email that file to your teams.
+
+This is a short list of what is possible.
+
+### Workflows flows
+In Workflows, you will create flows. A flow is a sequence of steps to complete a goal. It's similar to a script where every code line is a step to complete a goal. In Workflows, you will build the flow visually without writing any code. 
+
+You will learn about the Workflows building blocks in a few minutes. First, let's create your first flow. 
+
+## Getting access to Workflows
```

Comment:
Should this be Okta Workflows? We need to use the headers as opportunities to reinforce keywords

Reply:
Updated here and other headings.